create-cycle-app/index.js

```diff
@@ -88,10 +88,48 @@ function createApp (name, verbose, flavor) {
     }
   }
 
+  var testLibQuestion = {
+    name: 'testLib',
+    message: 'Which test library do you want to use?',
+    type: 'list',
+    choices: [
+      {
+        name: "Good ol' Mocha",
+        value: 'mocha'
+      },
+      {
+        name: 'Ava, the Futuristic JavaScript test runner',
+        value: 'ava'
+      }
+    ]
+  }
+
+  var cycleCoreLib = {
+    name: 'cycleLib',
+    message: 'Which cycle core library do you want to use?',
+    type: 'list',
+    choices: [
+      {
+        name: 'A basic Cycle',
+        value: 'cycle'
+      },
+      {
+        name: 'A super powerful Motorcycle',
+        value: 'motorcycle'
+      }
+    ]
+  }
+
   if (flavor) {
     // Ask just for the stream library
-    inquirer.prompt([streamLibQuestion]).then(function (answers) {
-      preparePackageJson(appFolder, appName, flavor, answers.streamLib, verbose)
+    inquirer.prompt([cycleCoreLib, streamLibQuestion, testLibQuestion]).then(function (answers) {
```

Comment:
It's a very good question. Considering recent efforts made by @Widdershin on https://github.com/cyclejs/time, we could consider expanding the default tests examples, showing the basics of how to test cycle components (at least for the current "Hello world" component)